### ISSUE TITLE
Code snipet on the readme seems wrong

### ISSUE THREAD

**Issue by bitcoinbrisbane** [orbitdb/orbitdb#1097]
Getting this error when trying to use the snippet off the readme "usage" section.

![image](https://github.com/orbitdb/orbitdb/assets/8411406/e94c9362-863e-4b8e-bbee-d1ec5674a515)

```javascript
import IPFS from 'ipfs-core'
import { createOrbitDB } from '@orbitdb/core'

;(async function () {
  const ipfs = await IPFS.create()
  const orbitdb = await createOrbitDB({ ipfs })

  // Create / Open a database. Defaults to db type "events".
  const db = await orbitdb.open("hello")
  
  const address = db.address
  console.log(address)
  // "/orbitdb/hash"
  // The above address can be used on another peer to open the same database

  // Listen for updates from peers
  db.events.on("update", entry => {
    console.log(entry)
    const all = await db.all()
    console.log(all)
  })

  // Add an entry
  const hash = await db.add("world")
  console.log(hash)

  // Query
  for await (const record of db.iterator()) {
    console.log(record)
  }
  
  await db.close()
  await orbitdb.stop()
})()
```

**Comment by Ahmedouu** [comment]
don't think we are getting the same error, but I agree the code there seems wrong or need more details about how to set it up

**Comment by bitcoinbrisbane** [comment]
> don't think we are getting the same error, but I agree the code there seems wrong or need more details about how to set it up

Let me try again ...

**Comment by aphelionz** [comment]
I believe that this block:

```javascript
// Listen for updates from peers
  db.events.on("update", entry => {
    console.log(entry)
    const all = await db.all()
    console.log(all)
  })
```

should be something like the following. Note the added `async`:

```javascript
// Listen for updates from peers
  db.events.on("update", async (entry) => {
    console.log(entry)
    const all = await db.all()
    console.log(all)
  })
```

**Comment by Ahmedouu** [comment]
if anyone gets this error 

import IPFS from 'ipfs-core'
^^^^^^
SyntaxError: Cannot use import statement outside a module

add "type":"module" to package.json and if you get another error like this:
SyntaxError: The requested module 'ipfs-core' does not provide an export named 'default'

Use: 
`import * as IPFS from 'ipfs-core' `

**Comment by silkroadnomad** [comment]
this was fixed here. https://github.com/orbitdb/orbitdb/pull/1113

**Comment by haydenyoung** [comment]
Closing issue as fixed. Feel free to re-open if the issue has not been resolved.
